.infrastructure/daemonset.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: todoapp-daemonset
+  namespace: mateapp
+spec:
+  selector:
+    matchLabels:
+      app: todoapp
+  template:
+    metadata:
+      labels:
+        app: todoapp
+    spec:
+      containers:
+      - name: todoapp
+        image: ikulyk404/busyboxplus:curl
+        resources:
+          requests:
+            memory: "100Mi"
+            cpu: "100m"
+          limits:
+            memory: "200Mi"
+            cpu: "200m"
+        command:
+        - sh
+        - -c
+        - while true; do echo "Sending request..."; curl -s http://todoapp-service:80; sleep 5; done
```

Comment:
Ensure that the service URL `http://todoapp-service:80` is correct and accessible from the DaemonSet pods. If the service is in a different namespace, you might need to specify the full service name including the namespace.